### ISSUE TITLE
Light bow duplication on invetory opening

### ISSUE THREAD

**Issue by ab67463** [Ormael7/Corruption-of-Champions#526]
Hello, 

I am running the latest 0.8s2. I have a strange bug - not sure when it happened, but I assume, that on equipping the Light bow.
![image](https://user-images.githubusercontent.com/94138153/141353876-9283755e-3b0b-4625-b31e-20850cd77d93.png)

The following happens - upon opening the inventory, I get an additional bow. 
![image](https://user-images.githubusercontent.com/94138153/141353983-d02e5a04-dafe-4796-b5a5-ff21c2bb279f.png)
This gets absurd pretty quickly, and obviously completely breaks the balance - basically free cash, just open the inventory a couple of times.

I am not sure if I can produce any logs, but just in case I saved to a file, not sure if that helps.
[BuggedSave.txt](https://github.com/Ormael7/Corruption-of-Champions/files/7522711/BuggedSave.txt)

**Comment by Ormael7** [comment]
Did you fighted with Wood Elfs party and had your range weapon disarmed?

**Comment by ab67463** [comment]
Yes, I was at one point disarmed, but I equipped the new, light bow only after the fight, when I should have gotten my old bow back.

**Comment by a1man57** [comment]
I'm having the same issue, I'm working with Xianxia_0.8s2 as well.

**Comment by Dragonborn20** [comment]
In case someone is struggling with this, I've found a temporary fix. Get rid of any light bows in your inventory (don't have it equipped and fill up your inventory so it doesn't dupe again), find the elves and make them disarm your ranged weapon, pick it up and finish the fight. At least for me that fixes the dupe.
